src/test/regress/expected/gporca.out

```diff
@@ -12210,9 +12210,9 @@ HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sur
          Hash Cond: (l1.lid = (meta.load_id)::integer)
          ->  Seq Scan on orca.t55 l1  (cost=0.00..13.00 rows=334 width=8)
                Output: l1.c, l1.lid
-         ->  Hash  (cost=0.02..0.02 rows=1 width=32)
+         ->  Hash  (cost=0.02..0.02 rows=1 width=0)
                Output: meta.load_id
-               ->  Subquery Scan on meta  (cost=0.00..0.02 rows=1 width=32)
+               ->  Subquery Scan on meta  (cost=0.00..0.02 rows=1 width=0)
```

Comment:
`width=0` doesn't seem to be correct - the tuple consists of the date and int columns with knows sizes.

Reply:
The width of zero came from `Result` node. This was mentioned in QA section.